docs/contributing.rst

```diff
@@ -0,0 +1,201 @@
+Contributing to Zarr
+====================
+
+Zarr is a community maintained project. We welcome contributions in the form of bug
+reports, bug fixes, documentation, enhancement proposals and more. This page provides
+information on how best to contribute.
+
+Asking for help
+---------------
+
+If you have a question about how to use Zarr, please post your question on
+StackOverflow using the `"zarr" tag <https://stackoverflow.com/questions/tagged/zarr>`_
+. If you don't get a response within a day or two, feel free to raise a `GitHub issue
+<https://github.com/zarr-developers/zarr/issues/new>`_ including a link to your StackOverflow
+question. We will try to respond to questions as quickly as possible, but please bear
+in mind that there may be periods where we have limited time to answer questions
+due to other commitments.
+
+Bug reports
+-----------
+
+If you find a bug, please raise a `GitHub issue
+<https://github.com/zarr-developers/zarr/issues/new>`_. Please include the following items in
+a bug report:
```

Comment:
This is definitely good. Should we link over to the issue template for more details?